src/api/Config/types.ts

```diff
@@ -202,9 +202,8 @@ export interface IAzureSearchConfig extends IVendorConfig {
 
 export interface IAlertsConfig {
   enabled?: boolean;
-  maxCasesLimit?: number;
+  maxMatchesLimit?: number;
```

Comment:
@mzorkany did you try it locally? You are missing the config updates.

Reply:
@davidrapin 
Sorry i forgot to mention in the description the server PR
https://github.com/Linkurious/linkurious-server/pull/3791

Reply:
why not move this change to the #3791 PR? these two PRs don't seem to make sense separately

Reply:
oh nevermind, different project, sorry